Core/Class.cs

```diff
@@ -0,0 +1,28 @@
+using System;
+
+namespace PCSharpGen.Core
+{
+    public class Class
+    {
+        // Maybe should be an enum?
+        public string Name { get; set; }
+
+        public int HitDie { get; set; }
+
+        public int SkillRanks { get; set; }
+
+        public float BaseAttackBonus { get; set; }
+
+        public float FortitudeSaveBase { get; set; }
```

Comment:
I'm not sure how we're planning to use Lua, but I think we could simplify these with some lambdas e.g.

```csharp
public Func<int, int> FortitudeSave { get; set; }
```

And then we could have a struct to represent the standard fast/slow save progressions:
```csharp
public struct SaveProgressions
{
	public static Func<int, int> FastProgression => (level => level / 2 + 2);
	public static Func<int, int> SlowProgression => (level => level / 3);
}
```

That would make your sorcerer declaration up top:
```csharp
Class sorceror = new Class
	{
		Name = "Sorceror",
		HitDie = 6,
		SkillRanks = 2,
		BaseAttackBonus = 1f/3f,
		FortitudeSave = SaveProgressions.SlowProgression,
		ReflexSave = SaveProgressions.SlowProgression,
		WillSave = SaveProgressions.FastProgression,
	};
```